tests/basic/longevity_test.go

```diff
@@ -1355,7 +1357,7 @@ func populateIntervals() {
 	triggerInterval[VolumeCreatePxRestart][2] = 9 * baseInterval
 	triggerInterval[VolumeCreatePxRestart][1] = 10 * baseInterval
 
-	baseInterval = 300 * time.Minute
+	baseInterval = 2 * time.Minute
```

Comment:
revert base interval of 2 min to original value

Reply:
fixed